src/app/mork/conf.py

```diff
@@ -61,6 +64,8 @@ class Settings(BaseSettings):
     EMAIL_FROM: str = "from@fun-mooc.fr"
     EMAIL_RATE_LIMIT: str = "100/m"
     EMAIL_MAX_RETRIES: int = 3
+    EMAIL_SITE_NAME: str = "FUN"
+    EMAIL_SITE_BASE_URL: str = "https://fun-mooc.fr"
```

Comment:
As those values are already defined through environment variable, IMO we should put empty values here in order to prevent mistakes during deployment. (e.g: if we forget to set those settings for the ademe website, FUN Mooc link will be used...)

Reply:
You're right!